include/eve/module/core/regular/bit_or.hpp

```diff
@@ -16,27 +16,24 @@ namespace eve
   template<typename Options>
   struct bit_or_t : strict_tuple_callable<bit_or_t, Options>
   {
-    template<eve::value T0, value T1>
-    requires(eve::same_lanes_or_scalar<T0, T1>)
-    EVE_FORCEINLINE constexpr bit_value_t<T0, T1>
-    operator()(T0 t0, T1 t1) const noexcept
+    template<value T0, value T1>
+    EVE_FORCEINLINE constexpr bit_value_t<T0, T1> operator()(T0 t0, T1 t1) const noexcept
```

Comment:
I really dislike `bit_value`
It has no documentation.
This name doesn't tell me anything. 

Our documentation for bit_or is also bad and doesn't specify a return value.

This is not for this pull request, I'm just voicing me being upset.

Reply:
Let fix this in the cleanup PR